### ISSUE TITLE
Crash during parsing file without sharedStrings

### ISSUE THREAD

**Issue by g3d** [mak-it/saxlsx#2]
File: https://www.dropbox.com/s/39k1pom4jq52xyy/students-10.xlsx?dl=0
Error:
```ruby
::Saxlsx::Workbook.open(import_file_path) {|w| w.sheets.first.rows[1] }
=> Errno::ENOENT: No such file or directory - xl/sharedStrings.xml
from /Users/g3d/.rvm/gems/ruby-2.1.1@isis/gems/rubyzip-1.0.0/lib/zip/file.rb:350:in `get_entry'
```
I tested rubyzip 1.1.0 and 1.0.0, the results is the same:
```ruby
::Saxlsx::Workbook.open(Import.first.import_file.path) {|w| w.sheets.first.rows[1] }
=> Errno::ENOENT: No such file or directory - xl/sharedStrings.xml
from /Users/g3d/.rvm/gems/ruby-2.1.1@isis/gems/rubyzip-1.1.7/lib/zip/file.rb:360:in `get_entry'
```

**Comment by ebeigarts** [comment]
MS Excel for Mac shows that this file has errors:

![screen shot 2015-03-26 at 11 37 13](https://cloud.githubusercontent.com/assets/8155/6843508/8c659e68-d3ac-11e4-8bc4-97fcb9fed5f7.png)

After repairing:

![screen shot 2015-03-26 at 11 38 08](https://cloud.githubusercontent.com/assets/8155/6843510/99817d42-d3ac-11e4-9611-0b5b040fbcef.png)

**Comment by g3d** [comment]
@ebeigarts Old one opened ok for me from Libreoffice. Updated link for correct ms excel for mac file, which still produce error during parsing by gem.

**Comment by ebeigarts** [comment]
This file uses inline strings (not shared strings), I released v1.1.0 that supports both types of strings.

**Comment by g3d** [comment]
Yes, new release fixed bug.
